frontend/js/pages/BattleDetail.js

```diff
@@ -13,34 +13,46 @@ function BattleDetail(props) {
   const { id } = useParams();
   const { battle } = props.battle;
   const { user } = props.user;
-
+  const fetchedBattle = props.location.query;
```

Comment:
where this is coming from? also what is this?

Reply:
This should be removed, please get the necessary information from the store

Reply:
Thanks for the advice! I'll fix this :)